index.html

```diff
@@ -42,5 +41,182 @@ <h5>LET'S CONNECT</h5>
                 </div>
             </div>
         </section>
+        <section class="works">
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot1.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Tonic</div>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
```

Comment:
- [ ] In this case, we should also use the correct HTML semantics tags. a `div` is simply a container element. but since this is text that is a description of the current project it would be much better if we used a `p` tag don't you think so.
Somthing like this would be much better.
```suggestion
                <p class="works-flex-item works-description">
                   A daily selection of privately personalized reads; no accounts or sign-ups required.
                </p>
```

- [ ] Please 🙏 implement this same change for all the other project cards.